edx_rest_framework_extensions/auth/jwt/tests/test_authentication.py

```diff
@@ -177,11 +177,16 @@ def test_authenticate_credentials_no_usernames(self):
     @mock.patch('edx_rest_framework_extensions.auth.jwt.authentication.set_custom_attribute')
     def test_authenticate_csrf_protected(self, mock_set_custom_attribute):
         """ Verify authenticate exception for CSRF protected cases. """
+
+        username = 'ckramer'
```

Comment:
why do we need this?

Reply:
We need to create a user to mock the response in: https://github.com/openedx/edx-drf-extensions/blob/a4c55012de9e7d762cf6dd0fe74f4bdb34107895/edx_rest_framework_extensions/auth/jwt/authentication.py#L67 The expected response is a tuple containing a user instance and data.
 It was previously mocked incorrectly as a tuple of strings.